### PR TITLE
Create catzboy.json

### PR DESCRIPTION
**Pull request by Maruful007**
Please add My Subdomain....

### CODE REVIEW

File: domains/catzboy.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Catzboy Profilo",
+  "repo": "https://github.com/Maruful007/Maruful007",
+  "owner": {
+    "username": "Maruful007",
+    "email": "marufulislamsami2007@gmail.com@gmail.com"
```
Comment (MaskDuck) [inline domains/catzboy.json:6]:
```suggestion
    "email": "marufulislamsami2007@gmail.com"
```

File: domains/catzboy.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  },
+  "record": {
+    "CNAME": [
```
Comment (MaskDuck) [inline domains/catzboy.json:9]:
```suggestion
    "CNAME": "https://maruful007.github.io/Catzboy-Profilo/public/",
```

File: domains/catzboy.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+      "https://maruful007.github.io/Catzboy-Profilo/public/"
```
Comment (MaskDuck) [inline domains/catzboy.json:10]:
```suggestion
```
delete this line pls

File: domains/catzboy.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    ]
```
Comment (MaskDuck) [inline domains/catzboy.json:11]:
```suggestion
```
this line also should be deleted

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by Maruful007** [comment]
is everything okay now??

File: domains/catzboy.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Catzboy Profilo",
+  "repo": "https://github.com/Maruful007/Maruful007",
+  "owner": {
+    "username": "Maruful007",
+    "email": "marufulislamsami2007@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://maruful007.github.io/Catzboy-Profilo/public/",
```
Comment (MaskDuck) [inline domains/catzboy.json:9]: cname shouldnt be an url

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/catzboy.json (hunk 5 above)
Comment (Maruful007) [reply domains/catzboy.json:9]: i want to catzboy.is-a.dev to look same as https://maruful007.github.io/Catzboy-Profilo/public/ what record should i put

**Comment by Maruful007** [comment]
done!!!
i hope no problem left...

**Review by MaskDuck — DISMISSED** [review]
not sus

**Review by TheBotlyNoob — DISMISSED** [review]
Looks good!

**Comment by Maruful007** [comment]
thx

**Comment by MaskDuck** [comment]
code owner said noice so yes

**Comment by MaskDuck** [comment]
what is the error

**Review by TheBotlyNoob — APPROVED** [review]
Okay, now we can merge it

**Comment by TheBotlyNoob** [comment]
The changes have been published!! It should reflect in less than 24 hours.

## Here's what you need to do next
If your domain points to a server you own, add `domain-name.is-a.dev` to your server config. For https, you will have to configure ssl certificate to allow the new subdomain.

### For github pages users,
* Go to your github page repo (`user/user.github.io`)
* Open up the **settings** tab
* Scroll down to the **Github pages** section
* In the **Custom domain** text input, enter the domain you registered (`domain-name.is-a.dev`)
* Check the **Enforce HTTPS** checkbox below the input
* Give it some time to reflect and you should be good to go

## Need help with your domain?
If you are having trouble setting up your domain, [create an issue](https://github.com/is-a-dev/register/issues/new/choose). I will try my best to get back to you asap!

## Made a mistake in the record?
Don't worry, you can create a new pull request with the corrections

## Love/Hate the service?
**Love it?** Leave it a **star**! Also consider donating so that I can keep this service running forever!

**Hate it?** Please leave your feedback by [creating an issue](https://github.com/is-a-dev/register/issues/new/choose). I would really like to keep improving this service for other users.

## Wanna support this project?

Help me in my mission to keep this service alive forever by donating!

<a href="https://www.buymeacoffee.com/phenax" target="_blank">Buy Me A Coffee</a> <a href="https://liberapay.com/phenax" target="_blank">Liberapay recurring donation button</a>

**Comment by Maruful007** [comment]
how do i request a subdomain....like project.catzboy.is-a.dev

**Comment by TheBotlyNoob** [comment]
just do the same thing, but just make the file called `project.catzboy.is-a.dev`

**Comment by Maruful007** [comment]
okk

**Comment by Maruful007** [comment]
brb

**Pull request merged**